### PR TITLE
Fix broken Yamllint Github Action

### LINKED ISSUE #284: yamllint GA abandoned not working since last year
The pipeline is failing every execution. I suggest we create replace the action and have single lint workflow with yamllint and ansible-lint as in:

https://github.com/staticdev/ansible-role-signal/blob/main/.github/workflows/tests.yml#LL16C1-L37C23

### PR DESCRIPTION
**Pull request by staticdev**
Closes #284 

This PR introduces following changes:
* Replaces broken yamlint GA with yamllint installation and execution steps
* Merges both yamllint and ansible-lint workflows for easier maintenance of linting

### CODE REVIEW

**Comment by staticdev** [comment]
@aalaesar @wiktor2200 I would like to merge this PR since lint part is working. Other errors are because of NC27 installation, it seems that now OCC changes/removed some argument we use on `php occ maintenance:install`.

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          pip install --constraint=.github/workflows/constraints.txt pip
+          pip --version
+
+      - name: Upgrade yamllint
+        run: |
+          pip install --constraint=.github/workflows/constraints.txt yamllint
+
+      - name: Lint code.
+        run: |
+          yamllint .
```
Comment (wiktor2200) [inline .github/workflows/lint.yml:38]: I think we should use config file that we have already prepared:
```suggestion
          yamllint -c .yamllint.yml .
```

**Comment by wiktor2200** [comment]
Hi @staticdev. Sorry for quite a long time, no response from my side.
Thanks a lot for keeping it up-to-date and actively maintained, I appreciate that :)
I've left small comment in code. But approving anyways if you agree with my comment just apply suggestion if not feel free to merge it as it is :)

**Review by wiktor2200 — DISMISSED** [review]
I've left small comment, but I think it's optional and can be changed later. Up-to you @staticdev :)

**Comment by staticdev** [comment]
> I've left small comment, but I think it's optional and can be changed later. Up-to you @staticdev :)

Thanks @wiktor2200, I added your change, now I need approval again XD

**Review by wiktor2200 — APPROVED** [review]

**Pull request merged**